components/Tabs/src/private/MDCItemBar.h

```diff
@@ -30,6 +31,15 @@
 /** Return the default height for the item bar given a style. */
 + (CGFloat)defaultHeightForStyle:(nonnull MDCItemBarStyle *)style;
 
+/**
+ Link back to this item bar's owning MDCTabBar so it can interrogate the tab bar's size class
+ delegate for a horizontal
+ size class.
```

Comment:
The linewrapping here seems off for the comment.

Reply:
Done, good catch!